projects/models.py

```diff
@@ -162,3 +187,26 @@ def post_create_setup(sender, instance, created, **kwargs):
 
         # Add "standard" components
         add_components_for_project(instance)
+
+        # Add catalog controls to project.
+        add_project_controls(instance)
+
+
+class ProjectControl(models.Model):
+    class Status(models.TextChoices):
+        NOT_STARTED = "not_started", _("Not started")
+        INCOMPLETE = "incomplete", _("Incomplete")
+        COMPLETE = "completed", _("Completed")
+
+    project = models.ForeignKey(to="projects.Project", on_delete=models.CASCADE)
+    controls = models.ForeignKey(to="catalogs.Controls", on_delete=models.PROTECT)
```

Comment:
Not a blocker, and definitely nitpicky:

Shouldn't "controls" be singular in this context?